.github/workflows/peer-api.yml

```diff
@@ -19,12 +19,8 @@ jobs:
       - name: Checkout
         uses: actions/checkout@v5
 
-      - name: Install lerna
-        run: npm install -g lerna@6.6.2
-
-      - name: Install semver
-        run: npm install -g semver
+      - name: Install script dependencies
+        run: npm ci --ignore-scripts
```

Comment:
Note: I did this here as a drive-by. Will do this where possible in a separate PR.